projects/kit/components/input-count/input-count-options.ts

```diff
@@ -14,12 +14,6 @@ export interface TuiInputCountOptions {
     readonly postfix: string;
 }
 
-/**
- * @deprecated: use TuiInputCountOptions instead
- * todo: remove in 3.0
- */
-export type InputCountOptions = TuiInputCountOptions;
-
 // TODO: 3.0 remove in ivy compilation
```

Comment:
Gotta not forget to go over all these fellas :D

Reply:
yes)) I'll to go over all related with "ivy compilation" after ng12 update